modules/backend/formwidgets/relation/partials/_relation.php

```diff
@@ -1,3 +1,3 @@
 <div class="relation-widget" id="<?= $this->getId() ?>">
-    <?= $this->makePartial('~/modules/backend/widgets/form/partials/_field_'.$field->type.'.php', ['field' => $field]) ?>
+    <?= $this->makePartial('~/modules/backend/widgets/form/partials/_field_' . $field->type . '.php', ['field' => $field]) ?>
 </div>
```

Comment:
@bennothommo do we have any rules for trailing newline on all files?

Reply:
Yeah I believe that's included in PSR12, but might not apply for the templates / partials.

Reply:
Can we add a rule to make it apply?

Reply:
It appears not - the code sniffer can only look at PHP blocks, it cannot read HTML as far as I can tell